### PR TITLE
S03E01 SUMMARY: add embedded pdf assets

### PR DESCRIPTION
**Pull request by CyFeng16**
https://github.com/BUPT/ai-ml.club/issues/255#issuecomment-520825623

### CODE REVIEW

File: docs/_posts/2019-08-11-seminar-meeting-minutes-3-1.md [hunk 1]
```diff
@@ -36,6 +36,30 @@ header:
...
+  
+  <div class="zoom-container" style="
+      position: relative;
+      padding-bottom:56.25%;
+      padding-top:30px;
+      height:0;
+      overflow:hidden;
+  ">
+    <iframe
+      src='{{ '/assets/js/viewer-js/#/assets/2019/seminar-3-1/oral-initc.pdf' | relative_url }}'
```
Comment (huan) [inline docs/_posts/2019-08-11-seminar-meeting-minutes-3-1.md]:
```diff
-       src='{{ '/assets/js/viewer-js/#/assets/2019/seminar-3-1/oral-initc.pdf' | relative_url }}'
+       src='{{ '/assets/js/viewer-js/' | relative_url }}#{{ '/assets/2019/seminar-3-1/oral-initc.pdf' | relative_url }}'
```

**Comment by huan** [comment]
Which chair is in charge of merging this PR?

Please follow the chairs' duty and help the PR to be merged, thanks!

**Comment by CyFeng16** [comment]
Sry, my bad. Forgot it. Coming soon.

**Comment by CyFeng16** [comment]
Dismiss this request due to the requested changes are outdated. Fix the bug manually instead.

**Comment by CyFeng16** [comment]
@huan PTAL. Many thanks!

**Pull request merged**